README.md

```diff
@@ -106,26 +106,28 @@ As you can see: thanks to all available RxSwift operators, in a simple way you m
 #### Waiting for proper BluetoothState
 In a following scenario: just after app launch, you want to perform scans. But, there are some problems with this approach - in order to perform work with bluetooth, you're manager should be in **.poweredOn** state. Specially for this case, our library provides you with another observable, that you should use for monitoring state.
 ```swift
-let stateObservable = manager.rx_state
+let stateObservable = manager.observeState()
 ```
-After subscribe, this observable will immediately emit next event with current value of BluetoothManager state, and later will fire every time state changes.
-You could easily chain it with operation you want to perform after changing to proper state. Let's see how it looks with scanning:
+This observable will emit next event with new value of BluetoothManager state every time state changes.
+You could easily chain it with operation you want to perform after changing to proper state. To get current bluetooth state, use `CentralManager.state`
+property. Let's see how it looks with scanning:
 ```swift
-manager.rx_state
+manager.observeState
+	.startWith(manager.state)
 	.filter { $0 == .poweredOn }
 	.timeout(3.0, scheduler)
 	.take(1)
```

Comment:
I think this `take(1)` is not needed. There won't be more than 1 event due to `ensure(.poweredOn` in `CentralManager`

Reply:
hmm, question is, it is possible that `CoreBluetooth` will send `poweredOn` twice?

Reply:
hmm, I think CoreBluetooth sends this state each time user re-enable bluetooth in his device

Reply:
yeah, but when it sends `poweredOn`, `poweredOff`, `poweredOn` then subscription is disposed due to `poweredOff` so `poweredOn` won't be called.
Problem might be when we will receive `poweredOn`, `poweredOn`, than without `take(1)` we will 2x start scanning.